DataManagementSystem/DB/FileCatalogComponents/DirectoryLevelTree.py

```diff
@@ -49,11 +49,13 @@ def findDir(self, path, connection=False):
   def findDirs(self, paths, connection=False):
     """ Find DirIDs for the given path list
     """
-    dpaths = ','.join(["'" + os.path.normpath(path) + "'" for path in paths])
-    dpaths = self.db._escapeString(dpaths)
-    if not dpaths['OK']:
-      return dpaths
-    dpaths = dpaths['Value']
+    dpathList = []
+    for path in paths:
+      dpath = self.db._escapeString(os.path.normpath(path))
```

Comment:
but if you do not put quotes around, your `in` query will be like `in (a,b,c)` instead of `in ('a', 'b', 'c')` no ?

Reply:
_escapeString itself adds quotes around the resulting string. This was actually creating problems